rnd/autogpt_server/autogpt_server/server/server.py

```diff
@@ -406,26 +312,37 @@ async def create_new_template(
         for link in graph.links:
             link.source_id = id_map[link.source_id]
             link.sink_id = id_map[link.sink_id]
-
-        return await autogpt_server.data.graph.create_graph(graph, is_template=True)
+        updated_graph = await autogpt_server.data.graph.create_graph(graph)
```

Comment:
Can we assume that if a new (non-template) graph version is created, it should be set as the new active version? If so, we could handle the deactivation of the current active version in `graph.create_graph`.